Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -356,6 +356,15 @@ void RenderBox::styleDidChange(StyleDifference diff, const RenderStyle* oldStyle
         }
     }
 
+#if ENABLE(DARK_MODE_CSS)
+    if (layer() && oldStyle && oldStyle->colorScheme() != newStyle.colorScheme()) {
+        if (auto* scrollableArea = layer()->scrollableArea()) {
+            scrollableArea->invalidateScrollCorner(scrollableArea->scrollCornerRect());
+            scrollableArea->invalidateScrollbars();
```

Comment:
This logic is similar enough to the logic added in `LocalFrameView::invalidateScrollbarsForAllScrollableAreas`, which makes me wonder if we should just fold `scrollableArea->invalidateScrollCorner(scrollableArea->scrollCornerRect())` into `ScrollableArea::invalidateScrollbars`, rather than having it written out in both places.

The only other call to `invalidateScrollbars()` is in `ScrollableArea::setScrollbarOverlayStyle`, so I think it would be fine to share the code.

Reply:
I'll do this locally and double check it works as expected (including the below comment)